crates/nu-command/src/platform/ansi/ansi_.rs

```diff
@@ -608,7 +608,12 @@ Format: #
             },
             Example {
                 description: "Use ansi to color text with a style (blue on red in bold)",
-                example: r#"$"(ansi -e { fg: '#0000ff' bg: '#ff0000' attr: b })Hello Nu World(ansi reset)""#,
+                example: r#"let format = {  # none of the following keys are mandatory, but any other key will be considered invalid
```

Comment:
I'm not sure that text is correct. Those are the only acceptable keys, but they're not all mandatory. e.g. I have this in my config and I think it works fine.
```
    leading_trailing_space_bg: { attr: n } # no fg, no bg, attr none, this effectively turns this off
```

Reply:
i like the text you propose, it's clearer :ok_hand:

Reply:
done in 66843de068786805794a3555c6fc0150a3cc2946 with the addition that any invalid keys will give an error :yum: